assets/static/filters.js

```diff
@@ -64,12 +68,65 @@ function setPause() {
     $(selectors.icon).removeClass("fa-circle-o-notch fa-spin fa-search").addClass("fa-pause");
 }
 
+function renderHistory() {
+    const storage = window.localStorage;
```

Comment:
It's an architectured dependency of this code, I'd really prefer if it could be provided, rather than assumed.